src/@types/declaration.d.ts

```diff
@@ -11,3 +11,7 @@ declare module "*.svg" {
   const value: any;
   export default value;
 }
+declare module "qrcode.react" {
```

Comment:
there is no type for this lib ? @types/qrcode.react ?

Reply:
im not sure how to handle types for modules. https://www.npmjs.com/package/qrcode.react

Reply:
if you want to check types exist => https://microsoft.github.io/TypeSearch/

Reply:
yes there is not type

Reply:
![image](https://user-images.githubusercontent.com/503185/73909044-7faddc80-48e6-11ea-98a8-96f9a00988de.png)

Reply:
oh no... i didn't see just now